src/sentry/static/sentry/app/views/organizationIntegrations/addIntegration.tsx

```diff
@@ -24,10 +33,14 @@ export default class AddIntegration extends React.Component {
     this.dialog && this.dialog.close();
   }
 
-  computeCenteredWindow(width, height) {
-    const screenLeft = window.screenLeft !== undefined ? window.screenLeft : screen.left;
+  dialog: any | null;
```

Comment:
If someone knows a better type for this let me know

Reply:
`Window | null`?